students/EricAdams/session02/series.py

```diff
@@ -0,0 +1,34 @@
+def fibonacci(n):
+    """ Return the fibonacci series nth value. """
+    if n == 0:
+        return 0
+    if n == 1:
+        return 1
+    return fibonacci(n-2) + fibonacci(n - 1)
+    
+
+def fib_series(n):
+    """ Print the nth value of the fibonacci series."""
+    print ('fibonacci(',n,') = ', fibonacci(n))
+    return 0
```

Comment:
Python will return None if you don't return anything -- and this is what you want if teh function doesn't return anything useful.